tools/c7n_left/requirements.txt

```diff
@@ -0,0 +1,11 @@
+cffi==1.15.1 ; python_version >= "3.7" and python_version < "4.0"
```

Comment:
is this supposed to be here? @kapilt

Reply:
cffi seems to be in other packages requirements.txt, and re this one it is part of how we interface to the golang runtime here, so it is appropriate.